### ISSUE TITLE
Help system polish and trimming

### ISSUE THREAD

**Issue by noahgibbs** [WebOfTrustInfo/prototype_vRWOT#13]
The help system, especially the tips system and initial online help, should be overridden by RWOT and replaced with something limited and appropriate. It should mention only verbs that RWOT actually supports well.

Where possible, other sources of online help should be updated (e.g. the tools popup hardcodes all *kinds* of outdated and weird help text.) But the priority should be on anything visible to end-users, with a secondary focus on anything visible to world-builders.

A lot of this is done in XML files (including the +tool popup, I think,) so I should see what parts can be repaired in the Tree of WOE. Those parts can potentially be handed off to Sarah, Will, etc.

**Comment by ChristopherA** [comment]
The first pass of this should probably be done by @saraht45, with an emphasis of writing a doc in the master SkotOS-Docs explaining how instances of SkotOS should do this (for instance, all the tips are in Woe, and can leverage acmds for auto-typing commands into Orchil), and then the specifics for vRWOT edited by @wip-abramson.

**Comment by saraht45** [comment]
**SkotOS:HelpNodes:***

**Remove** 
CNPCs, accent, books, catalog, character, chat, clothing, combat, courier (NIP), custom, dancing, death, dismiss, friends, fun, guard (NIP), injury, language, logging, mute, privacy, roleplaying, seamstress, skills, theme, AlterationHelp:*, Commands:address, Commands:dismiss, Commands:erase, Commands:learn, Commands:ooc, Commands:read, Commmands:recite, Commands:sign, Commands:study, Commands:teach, Commands:wear, Commands:remove, Commands:wield, Commands:write, Commands:assess, Commands:health, Commands:speak, SgCommands:!badname, SgCommands:!badnames, SgCommands:!dismiss, SgCommands:!drag, SgCommands:!goodname, SgCommands:!killnote, SgCommands:!mute, SgCommands:!notes, SgCommands:!office, SgCommands:!suspend, SgCommands:!swap, SgCommands:!unmute

**Keep**
accept, adverb, bulk, consent, containers, details, gait, loudness, movement, pose, proximity, revoke, room, stance, Commands:accept, Commands:approach, Commands:ask, Commands:assist, Commands:bug, Commands:close, Commands:eat, Commands:elude, Commands:enter, Commands:examine, Commands:exits, Commands:follow, Commands:forget, Commands:give, Commands:go, Commands:help, Commands:idea, Commands:inventory, Commands:leave, Commands:lock, Commands:look, Commands:more, Commands:offer, Commands:open, Commands:page, Commands:put, Commands:quit, Commands:refuse, Commands:remember, Commands:revoke, Commands:take, Commands:tip, Commands:touch, Commands:typo, Commands:who, Commands:whoami, SgCommands:!, SgCommands:!assist, SgCommands:!broadcast, SgCommands:!busy, SgCommands:!customers, SgCommands:!disconnect, SgCommands:!dismiss, SgCommands:!finger, SgCommands:!grab, SgCommands:!hotspots, SgCommands:!infoline, SgCommands:!join, SgCommands:!page, SgCommands:!people, SgCommands:!ungrab, SgCommands:!whereis

**Keep and update for vRWOT**
\_index\_, basics, bugs, clustering, communicate, concepts, credits, eating, emotion, gesture, newplayer, perception, manipulation, memory, rules, Commands:north

**Comment by saraht45** [comment]
**SkotOS:Tip** can be removed

**SkotOS:Tips:Intro**

**Remove**
23 - duplicate of 22

**Keep**
2 - help files
3 - communication
4 - look
5 - examine
6 - inventory
7 - movement
8 - exits
9 - verbs
10 - gestures
11 - adverbs
12 - adverbs list
13 - adverbs abbreviated
14 - proximity with objects
15 - proximity with people
16 - object interaction
17 - stances
18 - consent
19 - leaving proximity
20 - assists
22 - bugs, ideas, typos

**Keep but update for vRWOT**
1 - welcome (rewrite)
21 - online help (update to point to github docs)

**Add**
Jitsi information
Customizing chatter body

**Comment by noahgibbs** [comment]
Would removing SkotOS:Tip turn off the tips on login?

**Comment by saraht45** [comment]
No, it appears the login tip calls upon SkotOS:Tips:Intro, which is the same database the \@tip command uses.
